packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -112,8 +112,8 @@ export async function getComponentsPath(
   if (hasRootWorkspace()) {
     try {
       const workspaceRootPath = getRootWorkspacePath();
-      const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
-
+      let username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
+      username = username ? username : defaultUsernameOrAlias;
       if (isNullOrUndefined(username)) {
```

Comment:
With these changes the `if (isNullOrUndefined(username)) {` statement is not needed anymore.